Makefile

```diff
@@ -387,6 +389,27 @@ postgres-upgrade: ## initalize or upgrade postgres db to latest state
 	@$(MAKE_C) packages/postgres-database/docker build
 	@$(MAKE_C) packages/postgres-database/docker upgrade
 
+local_registry=registry
+.PHONY: setup-local-registry
+setup-local-registry: .env ## creates a local docker registry and configure simcore to use it (NOTE: needs admin rights)
```

Comment:
MINOR: using nouns instead of verbs sounds more readable to me, i.e. ``make local-registry`` . And shorter :-)

Reply:
done